app/src/test/java/com/example/habitsmasher/ui/history/HabitEventListTest.java

```diff
@@ -35,9 +37,67 @@ public void addHabitEvent_validHabitEventParametersAddition_ExpectHabitEventAdde
         String sampleComment = "Valid comment";
         Uri sampleUri = Uri.EMPTY;
 
-        _habitEventList.addHabitEvent(sampleDate, sampleComment, sampleUri);
+        _habitEventList.addHabitEventLocally(sampleDate, sampleComment, sampleUri, UUID.randomUUID().toString());
 
         assertEquals(1, _habitEventList.getHabitEvents().size());
         assertEquals(sampleComment, _habitEventList.getHabitEvents().get(0).getComment());
     }
+
+    @Test
+    public void deleteHabitEvent_existingEvent_ExpectHabitEventDeleted() {
+        HabitEvent event1 = new HabitEvent(new Date(), "Habit 1", UUID.randomUUID().toString());
+        HabitEvent event2 = new HabitEvent(new Date(), "Habit 1", UUID.randomUUID().toString());
+        HabitEvent event3 = new HabitEvent(new Date(), "Habit 1", UUID.randomUUID().toString());
+
+        _habitEventList.addHabitEventLocally(event1);
+        _habitEventList.addHabitEventLocally(event2);
+        _habitEventList.addHabitEventLocally(event3);
+
+        assertEquals(3, _habitEventList.getHabitEvents().size());
+        assertTrue(_habitEventList.getHabitEvents().contains(event1));
+        assertTrue(_habitEventList.getHabitEvents().contains(event2));
+        assertTrue(_habitEventList.getHabitEvents().contains(event3));
+
+        _habitEventList.deleteHabitEventLocally(event2);
+
+        assertFalse(_habitEventList.getHabitEvents().contains(event2));
+        assertEquals(2, _habitEventList.getHabitEvents().size());
+
+        _habitEventList.deleteHabitEventLocally(event1);
+        _habitEventList.deleteHabitEventLocally(event3);
+
+        assertFalse(_habitEventList.getHabitEvents().contains(event1));
+        assertFalse(_habitEventList.getHabitEvents().contains(event3));
+        assertEquals(0, _habitEventList.getHabitEvents().size());
```

Comment:
No concerns just wanted to say I love the whitespace here. Makes it a lot easier to read! 👍🏽